externs/olx.js

```diff
@@ -6527,12 +6539,22 @@ olx.tilegrid.TileGridOptions.prototype.tileSizes;
 
 
 /**
+ * Widths in tiles. If given, the array length should match the length of the
+ * `resolutions` array, i.e. each resolution will have a number of tile columns.
```

Comment:
Wouldn't it be good to explain when specifying this is necessary? (It's not clear to me for the moment.)